features/fixtures/fl-project/fastlane/Fastfile

```diff
@@ -23,10 +23,24 @@ platform :ios do
     upload_symbols_to_bugsnag(upload_url: endpoint, api_key: api_key, ignore_empty_dsym: ignore_empty_dsym, ignore_missing_dwarf: ignore_missing_dwarf)
   end
 
-  lane :upload_symbols_with_custom_action do
-    dsym_magic # Sets a custom value to DSYM_OUTPUT_PATH
-    upload_symbols_to_bugsnag(upload_url: endpoint, api_key: api_key)
+  lane :upload_symbols_after_gym do
+    # Sets a custom value to DSYM_OUTPUT_PATH
+    Actions.lane_context[SharedValues::DSYM_OUTPUT_PATH] = ["../app-center"]
```

Comment:
Is this value always set to an array?

Reply:
No, in fact it's never set to an array, it should be just a plain path according to the [docs](https://docs.fastlane.tools/actions/gym/#lane-variables), that needs fixing :)

On the flipside the SharedVariable of `download_dsyms` is always an array: https://docs.fastlane.tools/actions/download_dsyms/#lane-variables

Reply:
Fixed in [`ce9286f` (#67)](https://github.com/bugsnag/bugsnag-dsym-upload/pull/67/commits/ce9286f0dfd76d37f86e9efcf16de3412c8fa221)

Added new function to coerce into an Array if required. The `Dir` function should always return an array though, so not put this through the new function.